### DISCUSSION TITLE
conda Installation with biostar hand book

### DISCUSSION THREAD

**Discussion by akgreat22** [biostars/biostar-handbook-issues#234] [Q&A] [answered]
I'm quite new to bioinformatics and Biostar hand book has been a great help. I am using Ubuntu on windows 10. All installations worked but when I got to updating conda I got error message and I could not also create a ne environment. Below is the error message.

 conda update -y -n base conda
Collecting package metadata (current_repodata.json): failed

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/conda-forge/linux-64/current_repodata.json>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
'https://conda.anaconda.org/conda-forge/linux-64'

**Comment by ialbert** [comment, marked as answer]
This error has been seen before see the following discussion titled:

* [Troubleshooting: CondaHTTPError: HTTP 000 CONNECTION FAILED](https://github.com/biostars/biostar-handbook/discussions/185)
